From e2afcfb020fa16b465dd25d3455cd00487e68915 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=EC=86=90=EB=9F=89?= <loop.infinitely@gmail.com>
Date: Fri, 2 Feb 2018 19:21:58 +0900
Subject: [PATCH] build: BuildLuarocks.cmake: fix luacheck rockspec (#7961)

luacheck renamed the filename from luacheck-scm-1.rockspec to luacheck-dev-1.rockspec.
---
 third-party/cmake/BuildLuarocks.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/third-party/cmake/BuildLuarocks.cmake b/third-party/cmake/BuildLuarocks.cmake
index 76739593921..1343948d36d 100644
--- a/third-party/cmake/BuildLuarocks.cmake
+++ b/third-party/cmake/BuildLuarocks.cmake
@@ -152,7 +152,7 @@ if(USE_BUNDLED_BUSTED)
   # DEPENDS on the previous module, because Luarocks breaks if parallel.
   add_custom_command(OUTPUT ${LUACHECK_EXE}
     COMMAND ${LUAROCKS_BINARY}
-    ARGS build https://raw.githubusercontent.com/mpeterv/luacheck/3929eaa3528be2a8a50c593d687c8625205a2033/luacheck-scm-1.rockspec ${LUAROCKS_BUILDARGS}
+    ARGS build https://raw.githubusercontent.com/mpeterv/luacheck/master/luacheck-dev-1.rockspec ${LUAROCKS_BUILDARGS}
     DEPENDS busted)
   add_custom_target(luacheck
     DEPENDS ${LUACHECK_EXE})
